etc/omero.properties

```diff
@@ -952,6 +952,33 @@ omero.client.browser.thumb_default_size=96
 # certain situations.
 Ice.IPv6=1
 
+# Glacier2Template IceSSL defaults and overrides
+# see https://doc.zeroc.com/ice/3.6/property-reference/icessl
+# Any property beginning ``omero.glacier2.IceSSL.`` will be used to
+# update the corresponding IceSSL. property
+omero.glacier2.IceSSL=
+
+# Glacier2Template SSL maximum allowed protocol (mac bug)
+omero.glacier2.IceSSL.ProtocolVersionMax=tls1_1
+# Glacier2Template SSL allowed cipher suites
+omero.glacier2.IceSSL.Ciphers=ADH:!LOW:!MD5:!EXP:!3DES:@STRENGTH
+# Glacier2Template SSL verification requirements
+omero.glacier2.IceSSL.VerifyPeer=0
+# Glacier2Template SSL allowed protocols
+omero.glacier2.IceSSL.Protocols=tls1
+
+#############################################
+## Darwin (OS X) specific defaults for templated configuration files only.
+## ``darwin.xxx.yyy`` properties will override xxx.yyy properties in this file.
+## Do not set these properties in ``omero config``, instead set the original
+## ``xxx.yyy`` property
+#############################################
+darwin.omero.glacier2.IceSSL.Ciphers=(AES)
```

Comment:
Can I propose using `darwin` _suffix_ to match jvmcfg? https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/test/unit/test_jvmcfg.py#L82

Reply:
Can do, as long as we're sure they'll never be a third party property that ends in `.darwin`.

I think it messes up the hierarchy though:
- OMERO is running on a Darwin system.
- Glacier2 is run by OMERO
- IceSSL is a component of Glacier2
- Ciphers is an attribute if IceSSL

Reply:
> as long as we're sure they'll never be a third party property

Don't imagine we can ever be sure of that, though we can be sure we won't need these particular properties for ever. `omero.glacier2.darwin` would be an option, but it's likely hard to detect.

> I think it messes up the hierarchy though:

I understand, but it also seems odd to be doing basically the same type of override two different ways.

Reply:
How does this look?

Reply:
Thanks, and I like the idea of making the property "file-private" with the prefix. As long as that is made clear in the property name (perhaps also with the added benefit of getting across the fact, "this could change, please don't rely on it"), then :+1: from me. Other options that occurred to me while trying to evaluate the impact of this:

```
_darwin_omero
__darwin_omero
_omero...darwin
__omero...darwin
__omero_overrides...darwin
```